istioctl/cmd/root.go

```diff
@@ -98,13 +98,21 @@ func init() {
 	viper.SetDefault("xds-port", 15012)
 }
 
+func ConfigureLogging(cmd *cobra.Command, args []string) error {
+	if err := configureLogging(cmd, args); err != nil {
```

Comment:
why not `return configureLogging()` or make the existing function public?

Reply:
:+1: cleaned it up